### ISSUE TITLE
The command `send_test_notification` doesn't work

### ISSUE THREAD

**Issue by joeribekker** [open-zaak/open-zaak#667]
**Describe the bug**
Cannot sent a test notification.

**To Reproduce**
```
# docker exec -it openzaak-0 src/manage.py  send_test_notification
Traceback (most recent call last):
  File "/app/src/openzaak/management/commands/send_test_notification.py", line 27, in handle
    nrc_client.create("notificaties", data)
AttributeError: 'NoneType' object has no attribute 'create'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "src/manage.py", line 24, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/app/src/openzaak/management/commands/send_test_notification.py", line 36, in handle
    f"Something went wrong while sending the notification to {nrc_client.base_url}"
AttributeError: 'NoneType' object has no attribute 'base_url'
```

**Expected behavior**
Test notification is send.

**Comment by sergei-maertens** [comment]
#665 has the same root cause

-> the management command to configure Open Zaak takes a domain as input, but the API root configured for notifications lacks the `http(s)://` protocol. This breaks deriving the client.

Additionally, the command does not yet use `zgw_consumers.Service`, but uses the old `vng_api_common.APICredential`

**Comment by sergei-maertens** [comment]
Second aspect was fixed via #739

**Comment by joeribekker** [comment]
Probably due to a config error. Should raise a CommandError if nrc_client is None (meaning the configuration is incorrect).
